docs/c-grader/index.md

```diff
@@ -29,7 +29,7 @@ A full `info.json` file should look something like:
 
 ### `question.html`
 
-Most questions using this autograder will contain a `pl-file-editor` or `pl-file-upload` element, though questions using other elements (e.g., `pl-string-input` for short expressions) are also possible. The question should also include, in the `pl-submission-panel`, a `pl-external-grader-results` to show the status of grading jobs. It is also recommended to place a `pl-file-preview` element in the submission panel so that students may see their previous code submissions. An example question markup is given below:
+Most questions using this autograder will contain a `pl-file-editor` or `pl-file-upload` element, though questions using other elements (e.g., `pl-string-input` for short expressions) are also possible. The question should also include, in the `pl-submission-panel`, a `pl-external-grader-results` to show the status of grading jobs. It is also recommended placing a `pl-file-preview` element in the submission panel so that students may see their previous code submissions. An example question markup is given below:
```

Comment:
Absolutely not. To inform the rest of my review, did you review the changes that your tooling made?

Reply:
This was an oversight, sorry about that. I had disabled this rule but never fixed these 3 mentions that it fixed before it was disabled.

I'll self-review the grammar now.